libkbfs/conflict_resolver.go

```diff
@@ -374,7 +374,7 @@ func fileWithConflictingWrite(unmergedChains, mergedChains *crChains,
 // unmerged paths that need to be checked for conflicts later in
 // conflict resolution, for all subdirectories of the given path.
 func (cr *ConflictResolver) checkPathForMerge(ctx context.Context,
-	unmergedChain *crChain, unmergedPath path, unmergedChains *crChains,
+	unmergedChain *crChain, unmergedPath path, unmergedChains,
```

Comment:
I find it really confusing when the type of a param is split into the next line.  Can you please keep this as is, or moved `unmergedChains` down to the next line? (And all the places below.)